crates/kitsune_p2p/kitsune_p2p/src/spawn/actor/space.rs

```diff
@@ -412,7 +412,7 @@ impl SpaceInternalHandler for Space {
         let ro_inner = self.ro_inner.clone();
         let timeout = ro_inner.config.tuning_params.implicit_timeout();
         let fut =
-            discover::get_cached_remotes_near_basis(ro_inner.clone(), basis.get_loc(), timeout);
+            discover::search_remotes_covering_basis(ro_inner.clone(), basis.get_loc(), timeout);
```

Comment:
This was what was causing publishes to go too wide. According to the comment just above this, it was supposed to be cover nodes. I'm fairly sure that this was incorrect because 'near' is just an ordering by who is closest. Taking all 'near' nodes and partitioning them like this will mean anyone is fair game as the target of a delegated publish in a larger network.

Given that we don't recheck the basis when data arrives, this also means an increased validation load - which may well be related to what Holo are reporting on larger networks... Though it's hard to be sure and I'm not claiming that this is the only bug to be found :)